pkg/services/secrets/kvstore/helpers.go

```diff
@@ -29,6 +33,26 @@ func SetupTestService(t *testing.T) SecretsKVStore {
 	return kv
 }
 
+// Helpers to determine whether plugin startup failures are fatal to the app
```

Comment:
The helpers file are mostly related to testing, wouldn't be better to separate these functions that are used in runtime? Maybe add them to kvstore.go instead?

Reply:
yeah I can move it. I will rename helpers to test_helpers just for clarifications sake.